core/src/test/groovy/noe/server/AS7UndeployTest.groovy

```diff
@@ -0,0 +1,97 @@
+package noe.server
+
+import noe.eap.server.as7.AS7Properties
+import org.junit.AfterClass
+import org.junit.BeforeClass
+import org.junit.Test
+
+import static org.junit.Assert.assertFalse
+import static org.junit.Assert.assertNotNull
+import static org.junit.Assert.assertTrue
+
+class AS7UndeployTest {
+
+    private static final String as7Dir = "test-as7"
+
+    private static File baseDir
+    private static File deploymentsDir
+
+    private static AS7 testServer
+
+    @BeforeClass
+    static void prepare() {
+        baseDir = File.createTempDir('noe', 'ServerAbstract')
+        assertNotNull(baseDir)
+        deploymentsDir = new File(baseDir, "${as7Dir}/standalone/deployments")
+        deploymentsDir.mkdirs()
+        assertNotNull(deploymentsDir)
```

Comment:
@honza-kasik please invert these last 2 statements, checking for null after it's used doesn't make much sense :-)

Reply:
Thanks, inverted :)